README.md

```diff
@@ -1,7 +1,9 @@
 # dataplatform-probe
-Monitoring service for dataplatform services and events
+Monitoring service for dataplatform services and events  
 
-## Running
+This application continuously sends events to the dataplatform pipeline to test the latency of the pipeline.  
```

Comment:
And here.

Reply:
Yeah here its on purpose to cause the new sentence to show below